pkg/translator/prometheusremotewrite/helper_test.go

```diff
@@ -704,7 +704,7 @@ func TestPrometheusConverter_AddSummaryDataPoints(t *testing.T) {
 	}{
 		{
 			name:          "summary with start time",
-			isGateEnabled: false,
+			isGateEnabled: true,
```

Comment:
I think this `isGateEnabled` can simply be removed everywhere

Reply:
[eb4a0bf](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37399/commits/eb4a0bf8baef555971fed35def8aac46e29e01d9)